### ISSUE TITLE
Visual Studio 2017 Crashes once typing wrong package name in package.json

### ISSUE THREAD

**Issue by dongdong-yang** [microsoft/nodejstools#2110]
##### Steps to Reproduce
1. install anyone of packages, like: "jquery"
2. modify "jquery" to any non-existing package inside "package.json"
3. Visual studio crashes once if package not existing, before your typo input is saved

sometimes it's not crash, just need time to wait, 10-20 minutes. but VS is frozen exactly during the waiting time.

**Comment by amcasey** [comment]
@armanio123 Can you please check whether this still repros?

**Comment by armanio123** [comment]
@amcasey I tried on Visual Studio 2017 and 2019 Preview and I'm unable to reproduce. 

I used a "Blank Node.js Console Application" for tests and renamed `jquery` to `jqueryuiooiyuiuoy`.

the `npm` node looks like this:
![image](https://user-images.githubusercontent.com/13305542/57160905-fab5f580-6d9e-11e9-90af-1f2f401250be.png)

@fishingry have you tried updating Visual Studio? Is there a different step or `npm` node looks different than mine?
